app/js/components/answers/Index.react.js

```diff
@@ -46,12 +46,27 @@ class AllAnswers extends React.Component {
      Common.initTinyMceContent('.answer');
    }
 
+   sortAnswers(answers_obj){
+    let answers_arr = [], accepted_answer;
+    for (var key in answers_obj) {
+      answers_obj[key].accepted 
+      ? accepted_answer = answers_obj[key]
+      : answers_arr.push(answers_obj[key]);
+    }
+    answers_arr = answers_arr.sort((current, next) => {
+      return current.votes_count - next.votes_count;
+    })
+    answers_arr.push(accepted_answer);
+    return answers_arr;
+   }
+
    render () {
      var answers = [], keys=[];
-     if (!$.isEmptyObject(this.state.answers)) {
-       keys = Object.keys(this.state.answers)
+     let sorted_answers = this.sortAnswers(this.state.answers);
+     if (!$.isEmptyObject(sorted_answers)) {
+       keys = Object.keys(sorted_answers)
        for (var i = keys.length - 1; i >= 0; i--) {
```

Comment:
You mean looping through the object again? Technically, yeah. 
But we needed to complete the sorting of the answers before we start building react components based off them.